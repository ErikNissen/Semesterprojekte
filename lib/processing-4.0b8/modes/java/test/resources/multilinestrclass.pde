version https://git-lfs.github.com/spec/v1
oid sha256:d45e329b0a6d4b9d77ae3713940bd69850ee88af431a14823ddc471e7cde6acd
size 245
