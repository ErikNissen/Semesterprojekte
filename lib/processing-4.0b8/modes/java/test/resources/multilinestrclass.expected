version https://git-lfs.github.com/spec/v1
oid sha256:9217ef5cd3af50e7e06db443fb7c7e0d3019f030fee280af55779cce89b5bfd7
size 936
